### PR TITLE
Add the parameters for callback in the reservation API

### PR DESCRIPTION
**Pull request by BushraAsif**

### CODE REVIEW

**Review by shahbaz-mehar — COMMENTED** [review]
Changes reviewed.

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # Changelog
 All notable changes to this project will be documented in this file.
 
+## [unreleased]
```
Comment (emicha) [inline CHANGELOG.md:4]: @BushraAsif is this really expected?

**Comment by emicha** [comment]
@ss-altapay-com Could you approve and release when the threads are resolved?

**Comment by BushraAsif** [comment]
@ss-altapay-com, @oxeg, I have made changes to the release notes. Kindly review and consider merging [this](https://github.com/AltaPay/plugin-magento2-community/pull/128) pull request first, creating a tag for it, and then merging the current one. 
Thank you.

**Review by ss-altapay-com — APPROVED** [review]

**Pull request merged**